junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/BaseModel.java

```diff
@@ -6,9 +6,11 @@
 import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlValue;
 
-@XmlRootElement(name = "failure")
+/**
+ * Created by georgekankava on 5/5/16.
+ */
```

Comment:
for consistency reasons, please remove the author section
as all the source files do not include this sort of info.